src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CodeStyle/PreferBracesPreference.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#pragma warning disable CS1574 // XML comment has cref attribute that could not be resolved
```

Comment:
some comments that don't result when this is pulled into libs that don't reference all of the shared side of roslyn workspace.